projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/cdc/b2b/b2b-scenarios.core-e2e.cy.ts

```diff
@@ -188,15 +197,15 @@ describe('CDC B2B scenarios', () => {
       cdc.loginWithoutScreenSet(cdc.b2bUser.email, cdc.b2bUser.password);
     });
 
-    it('should display a new address form when no address exists', () => {
+    it.only('should display a new address form when no address exists', () => {
```

Comment:
I guess the PR is still work in progress. Before merging the PR, please make sure to remove `.only` specifiers.

Reply:
yes @Platonn, those were added as they were WIP. Will finalise them

Reply:
Understood. Feel free to mark this conversation as resolved, when the e2e tests are cleaned up from .only and .xit